src/chainstate/stacks/db/mod.rs

```diff
@@ -605,8 +589,12 @@ impl StacksChainState {
 
     /// Install the boot code into the chain history.
     /// TODO: instantiate all account balances as well.
-    fn install_boot_code<F>(chainstate: &mut StacksChainState, mainnet: bool, additional_boot_code_contract_names: &Vec<String>, additional_boot_code: &Vec<String>, f: F) -> Result<(), Error>
-    where F: FnOnce(&mut ClarityTx) -> () {
+    fn install_boot_code<F>(chainstate: &mut StacksChainState, 
+                         mainnet: bool, 
+                         additional_boot_code_contract_names: &Vec<String>, 
+                         additional_boot_code: &Vec<String>, 
+                         initial_balances: Option<Vec<(PrincipalData, u64)>>, f: F) -> Result<(), Error> where F: FnOnce(&mut ClarityTx) -> () {
```

Comment:
It doesn't have to be addressed in this PR, but can you open an issue to make it so `install_boot_code` uses a callback, like it originally does?  Eventually, when people launch their own testnets or app chains, their app chain implementation will provide its own boot code.

Reply:
The closure is still in place.

Reply:
Ah, lol, missed that.